plugin.json

```diff
@@ -7,7 +7,7 @@
     "license": "proprietary",
     "link": "http://store.shopware.com",
     "author": "Shopware AG",
-    "currentVersion": "1.0.2",
+    "currentVersion": "1.0.3",
```

Comment:
Damn... I forgot to increase the version number. The new version will be 1.0.4. Can you increase it?